src/DevAdventCalendarCompetition/DevAdventCalendarCompetition.Tests/IntegrationTests/TestStartup/StartupTestBase.cs

```diff
@@ -21,7 +21,10 @@ public static class StartupTestBase
         static StartupTestBase()
 #pragma warning restore CA1810 // Initialize reference type static fields inline
         {
+            var pathToAppsettingsDev = Path.GetFullPath(@"../../../../DevAdventCalendarCompetition");
```

Comment:
we want to add a referenced file to the test folder. In VS - option `Add existing item`
so this logic is not needed. We will have appsettings.Development.json in the test folder. And we need appsettings.Tests.json file (our custom file, only with DefaultConnection).

and logic here should be only
```
.AddJsonFile("appsettings.Development.json")
.AddJsonFile("appsettings.Tests.json"
```